src/plugins/intel_cpu/src/onednn/dnnl.cpp

```diff
@@ -148,5 +148,35 @@ unsigned get_cache_size(int level, bool per_core) {
     DNNL_THROW_ERROR(dnnl_unimplemented, "get_cache_size has no mode per_core == false");
 }
 
+unsigned get_cache_size_for_current_core(int level, bool per_core) {
```

Comment:
This function is very similar with `dnnl::impl::cpu::platform::get_per_core_cache_size` and  `dnnl::utils::get_cache_size()`, maybe mix them or other.
So could we implement this `get_cache_size_for_current_core` by calling above dnnl function?

Reply:
The dnnl function uses a static variable to get cpu cache size, and the initialization of this static variable can not be guaranteed to be performed on Pcore. So here we reacquire the cpu information in the current thread.